Translations/Localization.md

```diff
@@ -0,0 +1,108 @@
+> Localization describes the subsequent process of translating an internationalized theme/plugin. Localization is often abbreviated as l10n (because there are 10 letters between the l and the n.)
```

Comment:
```suggestion
> Localisation describes the subsequent process of translating an internationalised theme/plugin. It's often abbreviated as _l10n_ (because there are 10 letters between the _l_ and the _n_.)
```